tools/devpack.ps1

```diff
@@ -1,45 +1,74 @@
 ﻿param(
     [Parameter(Mandatory=$false)]
     [Switch]
-    $E2E
+    $E2E,
+    [Parameter(Mandatory=$false)]
+    [Switch]
+    $SkipBuildOnPack,
+    [Parameter(Mandatory=$false)]
+    [string[]]
+    $AdditionalPackArgs = @()
 )
 
 # Packs the SDK locally, and (by default) updates the Sample to use this package, then builds.
 # Specify --E2E to instead target the E2E test app.
 
 $rootPath = Split-Path -Parent $PSScriptRoot
 $project = "$rootPath/samples/FunctionApp/FunctionApp.csproj"
-$sdkProject = "$rootPath/sdk/Sdk/Sdk.csproj"
-$analyzerProject= "$rootPath/sdk/Sdk.Analyzers/Sdk.Analyzers.csproj"
+$sdkProject = "$rootPath/build/DotNetWorker.Core.slnf"
 
 if($E2E -eq $true)
 {
     $project = "$rootPath/test/E2ETests/E2EApps/E2EApp/E2EApp.csproj"
 }
 
+if ($SkipBuildOnPack -eq $true)
+{
+  $AdditionalPackArgs +="--no-build";  
+}
+
 $localPack = "$rootPath/local"
 if (!(Test-Path $localPack))
 {
   New-Item -Path $localPack -ItemType directory | Out-Null
 }
 Write-Host
 Write-Host "---Updating project with local SDK pack---"
-Write-Host "Packing SDK to $localPack"
-& "dotnet" "pack" $sdkProject "-o" "$localPack" "-nologo"
-Write-Host "Packing Analyzers to $localPack"
-& "dotnet" "pack" $analyzerProject "-o" "$localPack" "-nologo"
+Write-Host "Packing Core .NET Worker projects to $localPack"
```

Comment:
This should eventually be broken up into reusable functions. Keeping as is for now